gcn/notices/core/Statistics.schema.json

```diff
@@ -9,6 +9,14 @@
       "type": "number",
       "description": "False alarm rate: the rate of occurrence of non-astrophysical events that are of the same intensity or significance as the current event [Hz]"
     },
+    "net_count_rate": {
+      "type": "number",
+      "description": "Net count rate of the transient above the background [counts/s] over rate_duration and rate_energy_range. Do specify rate_duration and rate_energy_range with net_count_rate property."
+    },
+    "backgound_count_rate": {
+      "type": "number",
+      "description": "Count rate of the background for the transient [counts/s] over rate_duration and rate_energy_range."
```

Comment:
```suggestion
      "description": "Count rate of the background for the transient [counts/s] over rate_energy_range."
```

Reply:
@jracusin we can't use same rate_duration property for background. However we can add one for time range array for background.

Reply:
The background rate comes from using multiple intervals, but it is used to estimate the background during the source interval.  I think this use is fine.